### PR TITLE
Add select context menu

### PR DESCRIPTION
**Pull request by norascheuch**
We're gradually adding context menu actions for the new DB panel.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/package.json [hunk 1]
```diff
@@ -1004,6 +1012,10 @@
           "command": "codeQLDatabasesExperimental.setSelectedItem",
           "when": "false"
         },
+        {
+          "command": "codeQLDatabasesExperimental.setSelectedItemContextMenu",
+          "when": "false"
```
Comment (norascheuch) [inline extensions/ql-vscode/package.json:1017]: When does this evaluate? I can see the context action if 'false' is set and also when the whole line is removed..

**Review by shati-patel — APPROVED** [review]
Looks good, and works as expected locally 🎉 

Just one minor comment about `package.json` admin, and I've tried to answer your question (but shout if I can explain better!)

File: extensions/ql-vscode/package.json [hunk 2]
```diff
@@ -377,6 +377,10 @@
         "command": "codeQLDatabasesExperimental.setSelectedItem",
         "title": "✓"
       },
+      {
+        "command": "codeQLDatabasesExperimental.setSelectedItemContextMenu",
```
[the remark this answers is not in the document]
Comment (norascheuch) [reply extensions/ql-vscode/package.json:382]: 👍

**Comment by norascheuch** [comment]
Thanks for the explanation @shati-patel !

**Pull request merged**